admin-ui/src/components/settings/floating-save-button.tsx

```
@@ -26,21 +26,21 @@ export function FloatingSaveButton({
       )}
     >
-      <RainbowButton
+      <Button
         type="button"
         size="lg"
         onClick={onSave}
         disabled={!canSave}
-        className="shadow-2xl"
+        className="shadow-lg"
       >
         {saving
```

ESLint

[error] Insert `·?`

(prettier/prettier)